### DISCUSSION TITLE
Apollo Save Tool v1.8.7

### DISCUSSION THREAD

**Discussion by bucanero** [bucanero/apollo-ps3#138] [General]
### Added

* New cheat codes
  - Assassin's Creed II

### Fixed

* Fix PS2 Classics encryption/decryption progress bar

### Misc

* Update Web Server saves index page

<hr /><em>This discussion was created from the release <a href='https://github.com/bucanero/apollo-ps3/releases/tag/v1.8.7'>Apollo Save Tool v1.8.7</a>.</em>

**Comment by Coioteodg** [comment]
Oi.
Estou usando a versão mais recente deste aplicativo. Eu usei em outros jogos e foi tudo um sucesso, mas quando eu tentei aplicar os cheats em Resident operation raccon city, rage, e the evil within eu recebo uma mensagem de erro que os cheats não poderiam ter sido aplicados, seguido por uma mensagem que confirmou que o save foi alterado. No jogo, os cheats não foram aplicados.
Alguma correção para isso, ou está relacionada a um bug?

**Reply by Aleksandr-VL** [reply]
Thanks for the program! But there is an issue, the program doesn't work with multiple users on a single console. When I create a new user and try to run apollo program - my console just shows me a black screen and doesn't respond.

**Comment by bucanero** [comment]
> Thanks for the program! But there is an issue, the program doesn't work with multiple users on a single console. When I create a new user and try to run apollo program - my console just shows me a black screen and doesn't respond.

you need to provide more details, firmware version, HEN, CFW, etc. Apollo does work across multiple users and it has been tested, but your system could be returning unexpected data when the app needs to auto-detect values and failing to start.
